guides/v2.2/cloud/release-notes/cloud-tools.md

```diff
@@ -26,6 +26,8 @@ The `{{ site.data.var.ct }}` 2002.0.22 release changes the structure of the `{{
 
    -  {:.new}Updated the `composer.json` file for the `{{ site.data.var.ct }}` package to add a dependency for the `magento/magento-cloud-patches` v1.0.0 package.
 
+   -  {:.fix}<!--MAGECLOUD-4661-->Fixed an issue that caused the `{{ site.data.var.ct }}` patching process to fail when applying security-only patches for versions 2.3.2 and later releases. This issue was introduced by the new versioning scheme adopted for security-only patches in {{ site.data.var.ee }} v2.3.3. See [Security-only patches]({{ site.baseurl }}/guides/v2.3/release-notes/bk-release-notes.html#security-only-patches).
```

Comment:
Not `fail` but `skip patches, which were expected to apply, causing no patches to be applied at all`. This can be better re-worded, it's just the issue itself.

Also I would change `...security-only patches for versions 2.3.2 and later releases.` to `...2.3.2-p2 release and any future security-only releases.`

Let's call `security-only patches` as `security-only releases` to avoid confusion with actual patches (hotfixes), which we fixed. Also new scheme was adopted starting 2.3.2 with 2.3.2-p2 release.